apps/dashboard/app/javascript/path_selector/path_selector_data_table.js

```diff
@@ -168,7 +180,9 @@ export class PathSelectorTable {
   }
 
   setLastVisited(path) {
-    localStorage.setItem(this.storageKey(), path);
+    if(path) {
+      localStorage.setItem(this.storageKey(), path);
+    }
```

Comment:
I'm unable to replicate the error - but this should at least safeguard us from saving `undefined` and `null`.